fr/overview/proj4_overview.rst

```diff
@@ -0,0 +1,68 @@
+:Author: OSGeo-Live
+:Author: Mike Adair, Julien Moquet
+:Reviewer: Cameron Shorter, Jirotech
+:Version: osgeo-live11.0
+:License: Creative Commons Attribution 3.0 Unported (CC BY 3.0)
+
+.. (no logo) .. image:: ../../images/project_logos/logo-proj4.png
+..  :alt: project logo
+..  :align: right
+..  :target: http://proj4.org/index.html
+
+.. image:: ../../images/logos/OSGeo_incubation.png
+  :scale: 100 %
+  :alt: OSGeo Project
+  :align: right
+  :target: http://www.osgeo.org/incubator/process/principles.html
+
+Proj.4
+================================================================================
+
+Proj4 est une librairie qui fournit des outils pour transformer des points
+entre différents systèmes de coordonnées de référence. Un système de coordonnées géographique de référence permet de décrire tous les points de la Terre sous la forme d'un ensemble de coordonnées (comme la latitude, la longitude et l'élévation). Différents systèmes sont utilisés pour représenter les 3 dimensions de la Terre sur une carte plate de 2 dimensions.
+
+Proj4 est écrit en C++. Il est utilisé dans :doc:`MapServer <mapserver_overview>`, :doc:`GRASS GIS <grass_overview>`, :doc:`PostGIS <postgis_overview>`, :doc:`Mapnik <mapnik_overview>`, Thuban, OGDI, TopoCad, OGRCoordinateTransformation et bien d'autres. 
+ 
+Proj4 a été traduit du C dans un grand nombre d'autres languages (Voir `The group of Proj4 projects (MetaCRS) <http://trac.osgeo.org/metacrs/wiki>`_). Certains sont utilisables via l'interface en ligne de commande.
+Ces sous-projects incluent 
```

Comment:
Oui, d'accord, il y a encore des restes à corriger !
Yes, I agree, there's a bit more of MetaCRS to clean !